monitor/launch/launcher.go

```diff
@@ -104,6 +104,9 @@ func (nm *NodeMonitor) Run(sockPath string) {
 		}
 
 		pipe.Close()
+
+		// throttle breakage loops
+		time.Sleep(time.Second)
```

Comment:
I'd define a constant for this duration.

Reply:
excellent point. done